default.xml

```diff
@@ -14,16 +14,13 @@
     <project path="ikgt" name="ikgt-core" remote="intel" revision="trusty"/>
 
     <!-- op-tee projects -->
-    <project path="optee_os" name="optee_os"/>
-    <project path="optee_client" name="optee_client"/>
-    <project path="optee_test" name="optee_test"/>
-
-    <!-- toolchains projects -->
-    <project path="toolchain/gcc/x86_64-linux-android-4.9" name="platform/prebuilts/gcc/linux-x86/x86/x86_64-linux-android-4.9/" remote="aosp"/>
+    <project path="optee_os" name="optee_os" revision="3.13"/>
+    <project path="optee_client" remote="op-tee-linaro" name="optee_client"/>
```

Comment:
Do we need to specify revision for optee-client?

Reply:
The latest code can work, so don't need to checkout to tag 3.13 after downloading the latest code. If needs, we can  just update the wiki to notice the user to checkout to tag 3.13.